cli/lib/kontena/client.rb

```diff
@@ -515,9 +515,6 @@ def add_version_warning(server_version)
     # @return [Hash,Object,NilClass]
     def parse_json(json)
       JSON.parse(json)
-    rescue => ex
-      debug { "JSON parse exception: #{ex.class.name} : #{ex.message}" }
```

Comment:
Fixing this in #3240 to report a more useful error message. This also has some regression potential.